Compilation of main branch broken due to missing GuiRunner.hh

## Environment
* OS Version: both Ubuntu 18.04 and homebrew CI builds
* Source or binary build? source builds of the main branch as of https://github.com/ignitionrobotics/ign-launch/commit/64632e087de1687a3ed32a3dab6ba888559d7b83

## Description
* Expected behavior: compile successfully
* Actual behavior: compilation fails

## Steps to reproduce
<!-- Provide steps so we can try to reproduce this issue -->

1. Install ignition-fortress dependencies
2. Build ign-launch main branch from source
3. observer compilation error

## Output

* https://build.osrfoundation.org/job/ignition_launch-ci-main-bionic-amd64/14/console
* https://build.osrfoundation.org/job/ignition_launch-ci-main-homebrew-amd64/15/console

~~~
plugins/gazebo_gui/GazeboGui.cc:23:10: fatal error: 'ignition/gazebo/gui/GuiRunner.hh' file not found
#include <ignition/gazebo/gui/GuiRunner.hh>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [plugins/gazebo_gui/CMakeFiles/ignition-launch-gazebogui.dir/GazeboGui.cc.o] Error 1
~~~

the `GuiRunner.hh` header file was removed from the public API in https://github.com/ignitionrobotics/ign-gazebo/pull/875

--- a/plugins/gazebo_gui/GazeboGui.cc
+++ b/plugins/gazebo_gui/GazeboGui.cc
@@ -20,7 +20,6 @@
 #include <ignition/common/Console.hh>
 #include <ignition/common/Filesystem.hh>
 #include <ignition/gazebo/config.hh>
-#include <ignition/gazebo/gui/GuiRunner.hh>
 #include <ignition/gui/MainWindow.hh>
 #include "ignition/gazebo/gui/Gui.hh"
 #include "ignition/gazebo/gui/TmpIface.hh"

--- a/plugins/gazebo_gui/GazeboGui.cc
+++ b/plugins/gazebo_gui/GazeboGui.cc
@@ -22,7 +22,6 @@
 #include <ignition/gazebo/config.hh>
 #include <ignition/gui/MainWindow.hh>
 #include "ignition/gazebo/gui/Gui.hh"
-#include "ignition/gazebo/gui/TmpIface.hh"
 
 #include "GazeboGui.hh"
 

